pkg/kdump/kdump-client.es6

```diff
@@ -106,14 +106,18 @@ export class KdumpClient {
             path = "/var/crash";
 
         var dfd = cockpit.defer();
-        if (!target || target.target != "local") {
-            dfd.resolve();
-        } else {
+        if (target.target === "local") {
             // local path, try to see if we can write
             cockpit.script(testWritableScript, [path], { superuser: "try" })
                     .done(dfd.resolve)
                     .fail(() => dfd.reject(cockpit.format(_("Directory $0 isn't writable or doesn't exist."), path)));
+            return dfd.promise();
+        }
+        if (target.target === "nfs") {
+            if (!target.nfs.value.match(".+:.+"))
```

Comment:
This can be a little tighter. How about `\S+:/.+`? This ensures that the host has no spaces, and that the path is absolute.